Makefile

```diff
@@ -65,16 +65,16 @@ run-cmake-coverage:
 	cmake -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=$(PREFIX) -DCMAKE_RULE_MESSAGES=$(RULE_MESSAGES) -DMY_CXX_WARNING_FLAGS="--coverage" $(ADDITIONAL_CMAKE_OPTIONS) -S . -B coverage-build
 
 test/unittest: run-cmake-release
-	cmake --build build --target UnitTests -j $(CPU_CORES)
-	pushd build && ctest --output-on-failure && popd
+	cmake --build build --target unittest -j $(CPU_CORES)
+	pushd build && tests/unittest/unittest --output-on-failure && popd
 
 test/unittest-d: run-cmake-debug
-	cmake --build dbuild --target UnitTests -j $(CPU_CORES)
-	pushd dbuild && ctest --output-on-failure && popd
+	cmake --build dbuild --target unittest -j $(CPU_CORES)
+	pushd dbuild && tests/unittest/unittest --output-on-failure && popd
```

Comment:
Since we are passing these tests locally but not on CI, I have a suggestion to split these into individual commands and see a verbose output both on CI and locally.